cinema/serializers.py

```diff
@@ -2,0 +1,74 @@
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, Movie, MovieSession, CinemaHall
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+class GenreListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "name"
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = "__all__"
+
+
+class ActorListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = "first_name", "last_name"
```

Comment:
The `fields` attribute should be a tuple or list, not a single string. Change `fields = "first_name", "last_name"` to `fields = ("first_name", "last_name")` or `fields = ["first_name", "last_name"]`.